### PR TITLE
Updating rbd command list to reflect upstream

### LINKED ISSUE #777: [doc] 10.2  Mounting and Unmounting
### Type of report
   - [ ] bug
   - [ ] feature request

### Source Document
10.2  Mounting and Unmounting:

https://documentation.suse.com/ses/7/single-html/ses-admin/#storage-bp-integration-mount-rbd

### Summary
Can we replace all usage of `rbd map` to `rbd device map`, `rbd unmap` to `rbd device unmap`, and `rbd showmapped` to `rbd device list`? Although the old commands are still working, the new ones are preferred to use. See e.g. the upstream doc [1].

[1] https://docs.ceph.com/en/octopus/rbd/rbd-ko/

### PR DESCRIPTION
**Pull request by asettle**
Closes #777

### CODE REVIEW

**Comment by asettle** [comment]
@trociny I think that covers all cases, but let me know if the other commands such as `rbd resize` and `rbd snap` require changes.

File: xml/admin_rbd.xml [hunk 1]
```diff
@@ -369,27 +369,27 @@ data blocks changed from 2097152 to 2560000</screen>
...
        command:
       </para>
-<screen>&prompt.cephuser;rbd map <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> --PARAM1 VAL1 --PARAM2 VAL2</screen>
+<screen>&prompt.cephuser;rbd device map <replaceable>POOL_NAME</replaceable>/<replaceable>IMAGE_NAME</replaceable> --PARAM1 VAL1 --PARAM2 VAL2</screen>
       <para>
        In the following example you can see how to specify a user name and a
        keyring with a corresponding secret:
       </para>
-<screen>&prompt.cephuser;rbdmap map mypool/myimage id=<replaceable>rbd_user</replaceable>,keyring=/etc/ceph/ceph.client.rbd.keyring</screen>
+<screen>&prompt.cephuser;rbdmap device map mypool/myimage id=<replaceable>rbd_user</replaceable>,keyring=/etc/ceph/ceph.client.rbd.keyring</screen>
```
Comment (trociny) [inline xml/admin_rbd.xml:385]: The `rbdmap` comand should not be changed.

**Review by trociny — COMMENTED** [review]
othewise LGTM

File: xml/admin_rbd.xml (hunk 1 above)
Comment (asettle) [inline xml/admin_rbd.xml:385]:
```suggestion
<screen>&prompt.cephuser;rbdmap map mypool/myimage id <replaceable>rbd_user</replaceable>,keyring=/etc/ceph/ceph.client.rbd.keyring</screen>

**Review by trociny — APPROVED** [review]

**Pull request merged**